src/App.js

```diff
@@ -1,26 +1,12 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
```

Comment:
- [ ] All components should be class-based. Ensure that your App component is class-based just like the Calculator component.

Reply:
@DammyShittu Thanks I appreciate your precision